packages/@aws-cdk/aws-cloudfront/lib/origin.ts

```diff
@@ -205,3 +206,33 @@ function validateIntInRangeOrUndefined(name: string, min: number, max: number, v
     throw new Error(`${name}: Must be an int between ${min} and ${max}${seconds} (inclusive); received ${value}.`);
   }
 }
+
+/**
+ * Throws an error if custom header assignment is prohibited by CloudFront.
+ * @link: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/add-origin-custom-headers.html#add-origin-custom-headers-use-cases
```

Comment:
The correct anchor here is `#add-origin-custom-headers-denylist`, I believe. Or we can remove the anchor altogether; either way.

Reply:
Updated.